### ISSUE TITLE
Add to @ApiModelProperty a new parameter to specify that a property is hidden only in requests

### ISSUE THREAD

**Issue by spekdrum** [springfox/springfox#1976]
Sometimes in API docs you need to specify that an attribute should be hidden in a request but shown in a response. For example in autogenerated IDs.

It would be something like:

```
@ApiModelProperty( hiddenOnRequest = true )
private String id;
```

**Comment by MaksimOrlov** [comment]
Good evening, @spekdrum.

I think, there are the same topics about issue you asked ( #182, #563, #807, #895 and #1356.)

And there is a pull request to fix this #1671

**Comment by dilipkrish** [comment]
Closing as a duplicate of the model enhancements work. #1671
